<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QDialog" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>339</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>LoginForm</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QComboBox" name="box1">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:18px;
border-style: solid;
border-bottom:1px solid #5c5c5c;  


  
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdedit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid; 
border-color: red;
border-bottom:1px solid #5c5c5c;
font-size:16px;
 </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>190</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="set_btn">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>0</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
image:url(:/images/set.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="log_btn">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>230</y>
     <width>95</width>
     <height>44</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
color:white;
font-size:18
px;
background-color:#1f88d4;
</string>
   </property>
   <property name="text">
    <string>登 陆</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>358</width>
     <height>339</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/images/jyv1.1.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
